express-post-json/express-post-json/index.js

```diff
@@ -0,0 +1,26 @@
+const express = require('express');
+const app = express();
+let nextID = 1;
+const grades = {};
+
+app.get('/api/grades', (req, res) => {
+  const returnArray = [];
+  for (const key in grades) {
+    returnArray.push(grades[key]);
+  }
+  res.json(returnArray);
+});
+
+app.use(express.json());
+
+app.post('/api/grades', (req, res) => {
+  req.body.Id = nextID;
+  grades[nextID] = req.body;
+  res.json(req.body);
+  nextID++;
+  res.status(204);
```

Comment:
Setting the status after the response is sent won't do anything, you want to set the status then send the response, typically done on the same line. Also `204` means no content, here we want to use status `201` which means "created"